perfzero/lib/perfzero/report_utils.py

```diff
@@ -24,6 +25,32 @@
 from six import u as unicode  # pylint: disable=W0622
 
 
+def run_uploader_methods(uploader_methods, execution_summary):
```

Comment:
The method calls the user-provided method with the user-provided argument. The word `upload` is only known to the caller. How rename this method to e.g. `execute_method(method_path, *args, **kwargs)`?

Reply:
Done